### DISCUSSION TITLE
How to change rotation rate?

### DISCUSSION THREAD

**Discussion by Wr4Th-0f-D0g** [Sixze/ALS-Refactored#589] [Q&A] [answered]
I'm trying to make the character's aim rotation speed slower than the camera rotation for a bit of aim lag. So far I haven't had any luck figuring out how to change it. Any help would be appreciated.

**Comment by ameaninglessname** [comment]
https://github.com/Sixze/ALS-Refactored/discussions/401#discussioncomment-7484347

For movement:
![image](https://github.com/user-attachments/assets/1c7accce-fbe0-4f43-acbf-afe2309e1899)
`MS_Als_Normal`
`CF_Als_RotationInterpolationSpeed_Responsive`

For aim offset:
`UAlsAnimationInstance::RefreshView`
`UAlsAnimationInstance::RefreshSpine`

**Reply by Wr4Th-0f-D0g** [reply, marked as answer]
Yeah I looked at the various data assets but nothing seemed to make the change I wanted. I resorted to editing the spine interpolation in the c++
